docs/rule/kotlin_library.soy

```diff
@@ -0,0 +1,188 @@
+{namespace kotlin_library}
```

Comment:
There's already `__jvm_common.soy` that would be a good place to pull some of the common stuff out.